### ISSUE TITLE
Still mantained?

### ISSUE THREAD

**Issue by anthrolisp** [renerocksai/sublimeless_zk#112]
This is a great piece of software, really love it because I work on both Windows and Linux and it runs on both. I was just wondering if it's still being maintained. Thank you in advance!

**Comment by quesada** [comment]
Agreed, this is a fantastic piece of software! It'd be a pity if it went unmaintained

**Comment by bastiano** [comment]
2 years since the last update. Very sad time.

**Comment by damionx7** [comment]
It appears that Rene has left the development and moved on to obsidian
https://renerocks.ai/blog/why-obsidian/

**Comment by bastiano** [comment]
> It appears that Rene has left the development and moved on to obsidian
> https://renerocks.ai/blog/why-obsidian/

Good news!

**Comment by damionx7** [comment]
😅 why would that be good news ?

**Comment by bastiano** [comment]
>  why would that be good news ?

Because I use Obsidian :+1:

**Comment by damionx7** [comment]
Looks like obsidian will be a good replacement.
